OSRMTextInstructions.podspec

```diff
@@ -0,0 +1,47 @@
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.name = "OSRMTextInstructions.swift"
+  s.version = "0.0.1"
+  s.summary = "Project OSRM text instructions"
+
+  s.description  = <<-DESC
+  OSRM Text Instructions is a library for iOS, macOS, tvOS, and watchOS applications written in Swift or Objective-C that transforms OSRM route responses into localized text instructions.
+                DESC
+
+  s.homepage = "http://project-osrm.org/"
+
+  # ―――  Spec License  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.license = { :type => "ISC", :file => "LICENSE.md" }
```

Comment:
[The license](https://github.com/Project-OSRM/osrm-text-instructions.swift/blob/master/LICENSE.md) is actually BSD.